src/BuildScriptGenerator/DotNetCore/DotNetCorePlatformInstaller.cs

```diff
@@ -31,5 +33,22 @@ public virtual bool IsVersionAlreadyInstalled(string version)
                 dynamicInstallDir: Path.Combine(
                     this.CommonOptions.DynamicInstallRootDir, DotNetCoreConstants.PlatformName));
         }
+
+        public override void InstallPlatformSpecificSkeletonDependencies(StringBuilder stringBuilder)
+        {
+            stringBuilder.AppendLine($"echo 'Installing {DotNetCoreConstants.PlatformName} specific dependencies...'");
+
+            // .NET Core dependencies (this is universal for all versions of .NET Core)
+            stringBuilder.AppendAptGetInstallPackages(
+                "libc6",
```

Comment:
This is cool, so in this way we dynamic install these .NET dependencies when build and run .NET apps? Should we remove them from images.

Reply:
> This is cool, so in this way we dynamic install these .NET dependencies when build and run .NET apps?
Yes, we will dynamically install build dependencies for any of the languages, depending on what is detected. Its nice because we don't have to download the node dependencies if we are building ruby, etc.

> Should we remove them from images.
This is currently only enabled for CLI/skeleton images, see https://github.com/microsoft/Oryx/pull/1492/files#diff-9a1cdf9c4275ba1810b7e7647eda62ab2db40a59fd0c48be98b5386c6e5bf742R100-R103 This is because some of our pre-baked images can't use dynamic installation as they are being used in places that don't have open access to the internet.